importer/SeFornminSv.py

```diff
@@ -138,17 +139,6 @@ def set_location(self):
             self.add_statement("location", self.get_socken(
```

Comment:
I mean in `utils.get_wikilinks()` which in turn calls `mediawikiparserfromhell`. So if you send it a string without `[[` it should just return an empty list?